.changeset/orange-radios-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/astro': patch
+---
+
+Fixed turborepo cache to correctly store components directory
```

Comment:
@jacekradko I believe this is internal, so we don't need to notify developers about this change. We can have an empty changesets file

Reply:
I figured it would be good to force a release of `@clerk/astro` in case we published a version with missing `@clerk/astro/components`, but I can take it out

Reply:
Looks like latest has it, so I will empty out the changeset

Reply:
<img width="305" alt="Screenshot 2024-11-20 at 10 08 50 AM" src="https://github.com/user-attachments/assets/39d88940-32a2-455c-a2c5-ffd2aa9cce0a">

Reply:
Hmm, shall we do this then ?
```suggestion
---
'@clerk/astro': patch
---

Fix possible missing exported files from `@clerk/astro/components`
```